src/Desktop.Analyzers/UnitTests/DoNotUseInsecureDTDProcessingUseXmlReaderForSchemaReadTests.cs

```diff
@@ -0,0 +1,410 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
```

Comment:
The name of this file should mention "XmlTextReader" instead of "XmlReader". The point of the tests is that you're not safe unless you use an `XmlTextReader` whose `DtdProcessing` property is set to `Prohibit` (as opposed to its default value of `Parse`).